src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -43,7 +43,7 @@ public CommandResult execute(Model model) throws CommandException {
 
         for (Index targetIndex : targetIndexArray) {
             if (targetIndex.getZeroBased() >= lastShownList.size()) {
-                throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX + ": "
+                throw new CommandException(Messages.MESSAGE_PERSON_INDEX_OUT_OF_BOUND + ": "
```

Comment:
the final printed message is like `Error: The person index provided is out of bound: 100`, if you want to avoid the 2nd colon then maybe it can be rephrased like `Error: The person index 100 is out of bounds.` then can use format string to pass in the 100

Reply:
Thanks for suggesting, I think this is better!